backend/createHTML/viz.py

```diff
@@ -27,11 +27,15 @@
 start_ring = 0
 debug_mode =0
 
-if os.environ.get("ENV") == "DEV" or 'DEV' in sys.argv:
+os.chdir(sys.path[0]) # if this script is called from a different directory
+DEV = os.environ.get("ENV") == "DEV" or 'DEV' in sys.argv
+
+if DEV:
     path = ".." 
     rootPath = "../.."
-    deviceList = rootPath + "/dev-data/deviceList.json"
-    imgDST = rootPath + "/dev-data/temp"
+    deviceList = rootPath + "/dev/data/deviceList.json"
+    #imgDST = rootPath + "/dev/data/temp"
+    imgDST = rootPath + "/frontend/images"
```

Comment:
this is an attempt to bridge DEV/non-dev for non-destructive stuff. lets review that its ok